### PR TITLE
Support for go-errors

### PR DESCRIPTION
**Pull request by QuantumNightmare**

### CODE REVIEW

File: raygun4go.go [hunk 1]
```diff
@@ -175,10 +177,22 @@ func (c *Client) createPost(err error, stack stackTrace) postData {
...
+// Manually send a new error with the given message to Raygun. This will use the current execution stacktrace.
 func (c *Client) CreateError(message string) error {
 	err := errors.New(message)
 	post := c.createPost(err, currentStack())
+	
+	return c.submit(post)
+}
+
+// Manually send the given github.com/go-errors/errors.Error to Raygun. This will use the message and stacktrace of the given Error.
+func (c *Client) SendError(error *goerrors.Error) error {
```
Comment (NZKoz) [inline raygun4go.go]: You should probably make this just take an `error` and internally use conditional type checking to handle cases with the stack and cases which are just an error (glorified string) https://tour.golang.org/methods/15

Comment (QuantumNightmare) [reply raygun4go.go]: Thanks, I will do that

File: raygun4go.go [hunk 2]
```diff
@@ -175,10 +177,30 @@ func (c *Client) createPost(err error, stack stackTrace) postData {
...
+}
+
+// Manually send the given error to Raygun.
+// If the given error is a "github.com/go-errors/errors".Error, then its stacktrace will be used in the Raygun report.
+// For other errors, the current execution stacktrace is used in the Raygun report.
+func (c *Client) SendError(error error) error {
+	err := errors.New(error.Error())
+
+  var st stackTrace = nil
+  goerror := error.(*goerrors.Error)
```
Comment (NZKoz) [inline raygun4go.go]: believe you'll want the  form

```go
if goerror, ok := error.(*goerrors.Error); ok {
```

Comment (QuantumNightmare) [reply raygun4go.go]: Neat, thanks

**Pull request merged**